assets/sass/components/global/_googlesitekit-logo.scss

```diff
@@ -63,3 +63,23 @@
 		}
 	}
 }
+
+.googlesitekit-logo.googlesitekit-logo-mini {
+
+	.googlesitekit-svg-logo-g {
+		height: 19px;
+		width: 19px;
+	}
+
+	.googlesitekit-svg-logo-sitekit {
+		height: 17px;
+		margin-left: $grid-gap-phone / 4;
+		width: 78px;
+	}
+
+	.googlesitekit-svg-logo-sitekit-beta {
+		height: 17px;
+		margin-left: $grid-gap-phone / 2;
+		width: 102px;
+	}
+}
```

Comment:
This new CSS should also follow BEM as much as possible like the rest of this file:
* The new class should be `googlesitekit-logo--mini`.
* It should be nested in the original `.googlesitekit-logo` block.
* The three inner selectors that override the default properties should match the original selectors in the `.googlesitekit-logo` block (i.e. include the other variant, the original selectors are always pairs of two).

Reply:
-- in use now and the original selectors are repeated.